TS issue - Ambient module declaration cannot specify relative module name

Things are ok during development but I'm running into this issue when running `vue-tsc --noEmit` (as part of my build script).

I'm getting this issue:
```
node_modules/vite-plugin-icons/client.d.ts:9:16 - error TS2436: Ambient module declaration cannot specify relative module name.

9 declare module '/@vite-icons/*' {
```
It appears the TS compiler does not like declaring a module name starting with a "/"

Any ideas what's going on?

My tsconfig file:
```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "moduleResolution": "node",
    "strict": true,
    "baseUrl": "./",
    "jsx": "preserve",
    "sourceMap": true,
    "resolveJsonModule": true,
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "lib": ["esnext", "dom"],
    "types": ["vite/client", "node"],
    "paths": {
      "@/*": ["src/*"],
    }
  },
  "include": [
    "**/*.ts",
    "**/*.js",
    "src/**/*.tsx",
    "src/**/*.vue"
  ]
}
```

--- a/client.d.ts
+++ b/client.d.ts
@@ -5,9 +5,3 @@ declare module 'virtual:vite-icons/*' {
   const component: FunctionalComponent<SVGAttributes>
   export default component
 }
-
-declare module '/@vite-icons/*' {
-  import { FunctionalComponent, SVGAttributes } from 'vue'
-  const component: FunctionalComponent<SVGAttributes>
-  export default component
-}

